mod-orgs/examples/setting_post.sample

```diff
@@ -0,0 +1,4 @@
+{
+  "key": "BANKING_INFORMATION_ENABLED",
```

Comment:
Choosing between:
BANKING_INFORMATION_ENABLED: TRUE/FALSE
versus
BANKING_INFORMATION: ENABLED/DISABLED
WDYT?

Reply:
I think first option is more standardized and we can choose it, also lets value will be from lower case :
"BANKING_INFORMATION_ENABLED": true

Reply:
and collection.sample also should be corrected with lower case string in example

Reply:
done